docs/docs/recipes/transforming-data.md

```diff
@@ -54,3 +54,39 @@ export const query = graphql`
 
 - [Tutorial on transforming Markdown to HTML](/tutorial/part-six/#transformer-plugins) using `gatsby-transformer-remark`
 - Browse available transformer plugins in the [Gatsby plugin library](/plugins/?=transformer)
+
+## Transforming images into grayscale using GraphQL
+
+`gatsby-image` is a React component designed to work seamlessly with Gatsby’s [native image processing](https://image-processing.gatsbyjs.org/) capabilities powered by GraphQL and [gatsby-plugin-sharp] `/packages/gatsby-plugin-sharp/` to easily and completely optimize image loading for your sites.
```

Comment:
```suggestion
`gatsby-image` is a plugin designed to work seamlessly with Gatsby’s [native image processing](https://image-processing.gatsbyjs.org/) capabilities powered by GraphQL and [gatsby-plugin-sharp] `/packages/gatsby-plugin-sharp/` to optimize image loading for your sites.
```
The distinction here is a bit confusing. `gatsby-image` includes a react component, `Img`, that you can use. However, `gatsby-image` itself is a plugin.

Reply:
Should I also change that here https://www.gatsbyjs.org/docs/gatsby-image/

Reply:
Was just noticing that. I think that page could use some updating, but let's do that in a separate PR.

Reply:
Awesome!!